src/panels/lovelace/cards/hui-picture-elements-card.ts

```diff
@@ -77,16 +77,6 @@ class HuiPictureElementsCard extends LitElement implements LovelaceCard {
     }
 
     this._config = config;
-
-    this._elements = this._config.elements.map(
-      (elementConfig: LovelaceElementConfig) => {
-        const element = createStyledHuiElement(elementConfig);
```

Comment:
This should be fixed in a different way, if custom elements are not loaded yet, we will listen for them to be loaded. When it is loaded, the element will fire an event, `ll-rebuild` when that event is received the element should be recreated.

That recreate logic seems to be missing here, it should be the same as for a cards:
https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/views/hui-view.ts#L74-L84